go-controller/pkg/libovsdb/libovsdb.go

```diff
@@ -136,6 +136,10 @@ func NewSBClientWithConfig(cfg config.OvnAuthConfig, promRegistry prometheus.Reg
 	enableMetricsOption := client.WithMetricsRegistryNamespaceSubsystem(promRegistry,
 		"ovnkube", "master_libovsdb")
 
+	dbModel.SetIndexes(map[string][]model.ClientIndex{
+		sbdb.EncapTable: {{Columns: []model.ColumnKey{{Column: "chassis_name"}}}},
```

Comment:
one questions to that change: can there be existing leftover encaps for the same chassis_name but with different ips? If so, we may need to explicitly delete them before setting a new client index

Reply:
testing this PR locally I am seeing that old chassis encaps are cleaned up when the node ip is changed.

this is source and destination node encaps after changing source node IP 172.18..4 -> 172.18.255.254

source:

```
sh-5.2# ovn-sbctl list encap
_uuid               : 8e8adecc-b8f6-408a-9b8f-7bd2dffee37c
chassis_name        : "3dce988f-4195-45c0-9e6e-174d39abd704"
ip                  : "172.18.0.2"
options             : {csum="true"}
type                : geneve

_uuid               : 5a62ca06-9897-4a26-aa2e-e6327362f6d0
chassis_name        : "0a673b46-e8c7-4178-9b85-c11ca45f70e8"
ip                  : "172.18.255.254"
options             : {csum="true"}
type                : geneve

_uuid               : 3ccf1eab-5dc7-44b6-820d-a7279f176bf9
chassis_name        : "dda834ed-7ccd-4d3f-936f-bc912daefc06"
ip                  : "172.18.0.3"
options             : {csum="true"}
type                : geneve
```

destination:

```
sh-5.2# ovn-sbctl list encap
_uuid               : 409178bc-f6ad-497e-be0c-387e13044548
chassis_name        : "dda834ed-7ccd-4d3f-936f-bc912daefc06"
ip                  : "172.18.0.3"
options             : {csum="true"}
type                : geneve

_uuid               : f9b3430d-f238-47f0-9a4a-17284f3afce5
chassis_name        : "0a673b46-e8c7-4178-9b85-c11ca45f70e8"
ip                  : "172.18.255.254"
options             : {csum="true"}
type                : geneve

_uuid               : 2a3d4af6-8931-40ee-b435-e39dae7bebfd
chassis_name        : "3dce988f-4195-45c0-9e6e-174d39abd704"
ip                  : "172.18.0.2"
options             : {csum="true"}
type                : geneve
```

Reply:
also there cant be leftover encaps cause in 4.14 we go to IC so all new DBs are created. In the non-IC case there shouldn't be more than one encap per chassis, or odd things start happening with traffic flow.